### PR TITLE
hotfix/APPEALS-9584: Have LegacyTaskSerializer Set LegacyTask Instructions to an Empty Array by Default

### PR DESCRIPTION
**Pull request by ThorntonMatthew**
Resolves [APPEALS-9584](https://vajira.max.gov/browse/APPEALS-9584)

`LegacyTask`s do not have a default `instructions` value like their AMA counterparts. To address this, we will return an empty array for `instructions` from the `LegacyTaskSerializer` by default in order to have the output match the default value for AMA tasks.

## Acceptance Criteria:
- [ ] For any `LegacyTask`, including `JudgeLegacyDecisionReviewTask`s if the task 's appeal doesn't have attorney case review notes, the LegacyTaskSerializer should return an empty array for its `instructions` attribute.
- [ ] `LegacyTask`s should be able to be completed even if the task does not have any instructions.

## Testing Plan:
1. Open the Rails console and locate a LegacyAppeal:
```ruby
legacy_appeal = LegacyAppeal.first
```

2. Save the following user into a variable for later use:
```ruby
# Or any user from User.all.filter(&:attorney_in_vacols?)
attorney = User.find_by_css_id("BVASCASPER1")
```

3. Create an `AttorneyLegacyTask` on the appeal:
```ruby
leg_task_params = OpenStruct.new(
  vacols_id: legacy_appeal.vacols_id,
  date_due: 3.days.ago,
  assigned_to_location_date: 10.days.ago,
  created_at: 12.days.ago,
  docket_date: nil
)

leg_task = AttorneyLegacyTask.from_vacols(leg_task_params , legacy_appeal, attorney)
```

4. Log into Caseflow as `BVASCASPER1`.

5. Open your browser's developer tools (CTRL+SHIFT+I if on a Chromium-based browser) and open the network tab.

network-tab

6. Navigate to the Case Details page for the appeal. (`/queue/appeals/<legacy_appeal.external_id>`)

7. Refer to the Dev Tool's Network tab. Locate the request with the name of "task?role=Attorney". Select it, and then go to the "Preview" tab. Collapse each of the tasks in the JSON array and find the LegacyTask. Ensure that its `instructions` are `[]`.
legacy_task_network

```json
{
    "id": "3575931",
    "type": "legacy_task",
    "attributes": {
        "is_legacy": true,
        "type": "AttorneyLegacyTask",
        "assigned_on": "2022-08-25",
        "docket_name": "legacy",
        "docket_date": "2016-12-08",
        "docket_range_date": "2016-12-08",
        "appeal_id": 409,
        "user_id": "BVASCASPER1",
        "added_by_name": "Nash X Kuvalis",
        "added_by_css_id": "BVANKUVALIS",
        "task_id": "3575931-2022-08-25",
        "label": "Draft decision",
        "document_id": null,
        "work_product": null,
        "appeal_type": "LegacyAppeal",
        "timeline_title": "Decision drafted by attorney",
        "started_at": "2022-09-29T14:47:46.722-04:00",
        ".... a bunch of other": "key-value pairs ...",
        "latest_informal_hearing_presentation_task": {},
        "instructions": []  <------------- What to look at
    }
}
```

Previously this value would be left `null`, which caused errors upon trying to complete **any** task on the appeal.

8. Due to some VACOLS mocking constraints within the local environment it may be that in order to test completing a task with one of these LegacyTasks on the same appeal we will need to do so in UAT.

9. You may essentially test the end functionality by copying the JSON object from the Network tab and saving it as a variable in the console:

Network Tab:
copy-object

Console Tab:
tasks-const

Then run the snippet of code from `formatInstructions` that is causing the error in production:
```javascript
tasks.map((task) => console.log(task.instructions?.[1]))
```

### CODE REVIEW

File: spec/feature/queue/task_queue_spec.rb [hunk 1]
```diff
@@ -796,7 +796,7 @@ def validate_pulac_cerullo_tasks_created(task_class, label)
           ).click
           find("button", text: "Send case").click
           expect(page).to have_content(/Bob Smith.*'s case has been sent to the Confirm schedule hearing team/)
-          expect(appeal.tasks.pluck(:type)).to include(
+          expect(appeal.reload.tasks.pluck(:type)).to include(
```
Comment (ThorntonMatthew) [inline spec/feature/queue/task_queue_spec.rb:799]: This is an unrelated rspec feature test fix.

**Review by ajspotts — APPROVED** [review]
lgtm 🎉
